### ISSUE TITLE
openssl(1) section "PASS PHRASE ARGUMENTS"

### ISSUE THREAD

**Issue by petrovr** [openssl/openssl#13883]
A number of utilites refer to is - for example "For more information about the format of arg see the PASS PHRASE ARGUMENTS section in openssl(1).". There is no such section in 1.1+ manuals. Master is still not up to date.

**Comment by kroeckx** [comment]
It's called "Pass Phrase Options".

**Comment by petrovr** [comment]
Also it is new page in master.

**Comment by kroeckx** [comment]
I think we should actually change the pod files see that it's a
link to a section: L<manpage/section>. For example, where
it now says this in 1.1.1:

see the B<PASS PHRASE ARGUMENTS> section in L<openssl(1)>.

It should say:

see the L<openssl(1)/"Pass Phrase Options">.

It should output the correct text in that case, and we could check
that the section exists.

I expected the html output to actually contain a proper link to
the section, but it seems it doesn't. I'll file a bug for that.

**Comment by kroeckx** [comment]
> I expected the html output to actually contain a proper link to the section, but it seems it doesn't. I'll file a bug for that.

https://github.com/Perl/perl5/issues/18487

**Comment by petrovr** [comment]
Perhaps L<openssl(1)/"Pass Phrase Options"> should be fine for 1.1.1 .
For master it is "openssl-passphrase-options(1)", i.e. https://www.openssl.org/docs/manmaster/man1/openssl-passphrase-options.html

Both 1.1.1 and mastes still has references to "PASS PHRASE ARGUMENTS" about 20 in 1.1.1 and about 5 in master.

About openssl-passphrase-options(1) - It is not "General commands". It is outside command. I'm not convinced that section #1 is correct. It is more for section (5) File formats and conventions or may be (7) Miscellaneas.

**Comment by levitte** [comment]
Proposed fix in #13885.

As for the location of the "options" manuals, this was a subject of discussion #13315, you might want to read how we came to the final result.
In summary, I'd still argue that since this is about *command line* options, section 1 is still the most appropriate, and we chose the file name 'openssl-XXX-options' to make it explicit that these are options pages.

The trouble with the Unix manual sections is that they are quite a coarse structure, and leave much up to interpretation.
(there are worse horrors, btw...  perlfaq(1), perlfunc(1), ...)
